Network/src/Game/game_tick.c

```diff
@@ -0,0 +1,52 @@
+/*
+** EPITECH PROJECT, 2024
+** Zappy_Server
+** File description:
+** No file there , just an epitech header example .
+*/
+
+#include "Server/server.h"
+#include <sys/time.h>
+
+static uint64_t get_time(void)
+{
+    struct timeval tv;
+
+    gettimeofday(&tv, NULL);
+    return tv.tv_sec * (uint64_t)1000000 + tv.tv_usec;
+}
+
+static void spawn_resources(server_t *server)
+{
+    int x;
+    int y;
+
+    for (int k = 0; k < MAX_ITEMS; k++) {
+        for (int i = 0; i < server->game->picked_up_items[k] &&
+        server->game->picked_up_items[k] > 0; i++) {
+            x = rand() % server->info_game.width;
+            y = rand() % server->info_game.height;
+            server->game->map[x][y].inventory[k]++;
+        }
+        if (server->game->picked_up_items[k] > 0) {
+            server->game->picked_up_items[k] = 0;
+        }
```

Comment:
remove the curly bracket 

```suggestion
        if (server->game->picked_up_items[k] > 0)
            server->game->picked_up_items[k] = 0;
```

Reply:
why should i ?